editor/components/NotesModal.tsx

```diff
@@ -2,16 +2,15 @@ import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
 import { useForm, SubmitHandler } from "react-hook-form";
-import { CrashNote } from "@/types/crashNote";
-import { INSERT_CRASH_NOTE } from "@/queries/notes";
 import { useMutation } from "@/utils/graphql";
 import { useAuth0 } from "@auth0/auth0-react";
 
 interface NotesModalProps {
   show: boolean;
-  onClose: () => void;
-  onSubmitCallback: (data: CrashNote) => void;
-  crashPk?: number;
+  handleCloseModal: () => void;
+  onSubmitCallback: () => void;
```

Comment:
i think its okay that im updating this callback to take no arguments?

Reply:
seems like it!